improver_tests/utilities/test_load.py

```diff
@@ -143,7 +143,7 @@ def test_multiple_constraints(self):
         realization_points = np.array([0])
         longitude_points = np.array([0])
         constr1 = iris.Constraint(realization=0)
-        constr2 = iris.Constraint(longitude=lambda cell: -0.1 < cell < 0.1)
```

Comment:
Is this needed because `cell` is the whole coordinate (points and bounds)?  Is it possible to specify the constraint on the point?  Is that a bad idea?

Reply:
The iris documentation says that [coordinate level constraints can be of several types](https://scitools.org.uk/iris/docs/latest/iris/iris.html?highlight=constraint#iris.Constraint), the type used being "a function which accepts a iris.coords.Cell instance as its first and only argument returning True or False if the value of the Cell is desired". The other types are either a single string, int or float to match or a list of values that the coordinate may match. With these types, passing values of anywhere between -4.9 and 5 will allow the test to pass as the values fall within the cell bounds. So it could be tested in this way.

It was originally using the callable type so I left as is as I'm unsure if this method was chosen for a reason, maybe to demonstrate different types of constraints as `constr1` uses a single value?

Reply:
Ah I see.  This documentation is not fun!  However I infer that "cell" includes points and bounds, and when you create the cell and use comparisons like the above it returns false if the cell bounds are not completely within the range specified.

To recreate the original comparison I think you could use either of: `longitude=lambda cell: -0.1 < cell.point < 0.1` or `longitude=lambda cell: cell.contains_point(0)`.  Would you mind trying those?  If neither works it's OK as it is.

Reply:
Both worked, thank you. Updated [here](https://github.com/metoppv/improver/pull/1309/commits/1d7f21003ca70fbe79b8c8943e280bc37f411b98#diff-2fb56b437f8e4077d745811cfdf984daL143).